docs/technical-details/self-hosted/ingress.md

```diff
@@ -27,3 +27,44 @@ ingress:
     nginx.ingress.kubernetes.io/proxy-body-size: 24m
 ```
 
+## Self-signed SSL Certificate
+
+This section applies if the Insights Ingress uses a custom SSL certificate which is not chained to any intermediate or root Certificate Authority. These instructions do not apply if using an SSL certificate provided by Letsencrypt or another recognized certificate issuer.
+
+### Self-signed SSL Certificate Validation - Insights Agent
+
+The [Insights agent](/configure/agent/configuration) needs to be provided a certificate to validate communication with the Insights API. This can be accomplished by creating a Kubernetes Secret containing the certificate, then including Helm values that intruct the agent chart to use that certificate.
```

Comment:
should this be instruct instead of intruct?

"then including Helm values that intruct the agent chart"

Reply:
Ah yes, thank you.